lib/data/replacements.json

```diff
@@ -751,6 +751,15 @@
       }
     ]
   },
+  "mvdan-sh": {
+    "description": "`mvdan-sh` uses unmaintained `gopherjs` so it is very outdated.",
+    "packageRules": [
+      {
+        "matchDatasources": ["npm"],
```

Comment:
This is much too wide of a predicate. We need a `"matchPackageNames"` condition on it as well.

Reply:
Isn't the key name as the original package name? Sorry I'm not familiar with this code base yet.

Reply:
No, you need to specify the original package names that you want to replace explicitly.

The key name convention appears to be "<orignal name>-to-<new name>", so in this case "mvdan-sh-to-sh-syntax".

Reply:
@astellingwerf I did a rebase and addressed that!